test/api/api.js

```diff
@@ -29,7 +29,12 @@ server.route({
   path: '/socket/start',
   handler: function (request, reply) {
     var filePath = path.join(__dirname, 'web-socket.js');
-    socketProcess = spawn('node', [filePath]);
+    socketProcess = spawn('node', [filePath], { stdio: 'inherit' });
+
+    filePath = path.join(__dirname, 'link.js');
```

Comment:
Rather than changing filePath after spawning the process, I would find it easier to read if there was both `socketPath` and `linkPath` definitions. Thoughts?

Also, yesterday you mentioned why the link process needed to be executed after the socket process. Could you include that as a quick comment here.